src/modules/ListTasks.js

```diff
@@ -0,0 +1,174 @@
+    checkBok.addEventListener('change', (event) => {
+      const { id } = event.currentTarget.parentElement.parentElement;
+      if (event.currentTarget.checked) {
+        event.currentTarget.nextSibling.style.textDecoration = 'line-through';
+        this.list[id - 1].completed = true;
+      } else {
+        event.currentTarget.nextSibling.style.textDecoration = 'none';
+        this.list[id - 1].completed = false;
+      }
+    });
+
+    if (task.completed) {
+      checkBok.click();
+      inTaskDescription.style.textDecoration = 'line-through';
+    }
+
+    inTaskDescription.addEventListener('focus', (event) => {
+      const { id } = event.currentTarget.parentElement.parentElement;
+      document.getElementById(id).style.backgroundColor = 'rgb(156, 156, 255)';
+      const iDelete = event.currentTarget.parentElement.nextSibling;
+      const iMenu = event.currentTarget.parentElement.nextSibling.nextSibling;
+      iDelete.classList.remove('hidden');
+      iMenu.classList.add('hidden');
+    });
+
+    /* Update value when stop focus */
+    inTaskDescription.addEventListener('focusout', (event) => {
+      const { id } = event.currentTarget.parentElement.parentElement;
+      const litask = document.getElementById(id);
+      litask.style.backgroundColor = 'white';
+
+      this.list[id - 1].description = event.currentTarget.value;
+      const iDelete = event.currentTarget.parentElement.nextSibling;
+      const iMenu = event.currentTarget.parentElement.nextSibling.nextSibling;
+
+      /* Set time out before hide the iDelete:
+      to make sure the click event always be tiggered before focusout */
+      setTimeout(() => {
+        deleteHid();
+      }, 200);
+    });
+
+    /* Display element */
+    divTaskContent.append(checkBok);
+    divTaskContent.append(inTaskDescription);
+
+    const iMenu = document.createElement('i');
+    iMenu.classList.add('fa-solid', 'fa-ellipsis-vertical');
+    iMenu.setAttribute = ('onclick', 'deleteHid()');
+
+    const iDelete = document.createElement('i');
+    iDelete.classList.add('fa-solid', 'fa-trash', 'hidden');
+    iDelete.style.color = '#fff';
+    iDelete.setAttribute = ('onclick', 'deleteHid()');
+    iMenu.addEventListener('click', (event) => {
+      deleteHid();
+      iDelete.style.color = '#c1c1c1';
+    });
+    iDelete.addEventListener('click', (event) => {
+      const { id } = event.currentTarget.parentElement;
+      this.delete(id); // Note: is id = index + 1
+    });
+
+    liTask.append(divTaskContent);
+    liTask.append(iDelete);
+    liTask.append(iMenu);
+
+    function deleteHid() {
+      iMenu.classList.toggle('hidden');
+      iDelete.classList.toggle('hidden');
+    }
+
+    this.addEventsDragAndDrop(liTask);
+
+    ulListTask.append(liTask);
+  }
+
+  delete = (idDeleted) => {
+    this.list = this.list.filter((task) => task.index !== this.list[idDeleted - 1].index);
+    document.getElementById(idDeleted).remove();
+    this.updateIndexs();
+  }
```

Comment:
- [ ] The same on deleted and update books 👍🏻 Here is a screenshot, your UI only shows two books but your local store has five books. Please correct this.

![image](https://user-images.githubusercontent.com/79058364/236284772-fd143b1f-00b7-46b4-8446-f1c0ae323058.png)